index.html

```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="sec-grid-container">
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="last-grid-container">
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
```

Comment:
- [ ] Please consider wrapping all these elements into one work `section`. This will make your code look much cleaner 🧼 and maintainable 😄